packages/framework/src/handler.ts

```diff
@@ -262,6 +297,9 @@ export class NovuRequestHandler<Input extends any[] = any[], Output = any> {
       }
 
       return this.createError(error);
+    }
+    if ((error as any).response.status === HttpStatusEnum.BAD_REQUEST) {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** Could we chain this into an `else if` for consistency with the existing implementation?

‏
```suggestion
    } else if ((error as any).response.status === HttpStatusEnum.BAD_REQUEST) {
```

Reply:
Definitely, it should have been else if my miss, thanks for rasing it up!